build/source/driver/multi_driver.f90

```diff
@@ -266,7 +288,7 @@ program multi_driver
 ! nGRU-is the total number of GRUs of the simulation domain
 ! nHRU-is the total number of HRUs of the simulation domain
 ! hruCount-is a local variable for the total number of HRUs in a GRU
-call allocate_gru_struc(nGRU,nHRU,err,message); call handle_err(err,message)
+call allocate_gru_struc(nGRU,nHRU,strtHRU,err,message); call handle_err(err,message)
```

Comment:
It might be cleaner to separate the allocate from the assign. I have not thought deeply about this like Andy has, but this does seem a little strange to pass a start index to an allocate routine.